tests/Integration/Controller/ListControllerTest.php

```diff
@@ -107,6 +108,16 @@ public function getListsWithCurrentSessionKeyReturnsListData()
                     'public' => true,
                     'category' => '',
                     'id' => 2,
+                ],
+                [
+                    'name' => 'Tech news',
+                    'description' => '',
+                    'creation_date' => '2019-02-11T15:01:15+00:00',
+                    'list_position' => 12,
+                    'subject_prefix' => '',
+                    'public' => true,
+                    'category' => '',
+                    'id' => 3,
                 ]
```

Comment:
While you're at id, could you please add the comma after the last array element which I had forgotten?